DNN Platform/Library/Security/PortalSecurity.cs

```diff
@@ -280,20 +328,45 @@ public static bool IsInRoles(UserInfo objUserInfo, PortalSettings settings, stri
             return isInRoles;
         }
 
+        /// <summary>
+        /// Determines whether the specified user is a friend of the current user.
+
+        /// </summary>
+        /// <param name="userId">The user identifier.</param>
+        /// <returns>
+        ///   <c>true</c> if the specified user is a friend of the current user; otherwise, <c>false</c>.
+
+        /// </returns>

```

Comment:
```suggestion
        /// <param name="userId">The user identifier.</param>
        /// <returns>
        ///   <c>true</c> if the specified user is a friend of the current user; otherwise, <c>false</c>.
        /// </returns>
```